### DISCUSSION TITLE
No changes but library calls responding 400 / invalid argument

### DISCUSSION THREAD

**Discussion by deadhead-pharaoh** [sigma67/ytmusicapi#677] [General]
Have been running my code for months without issue, working fine until last week.  Today I get 400 bad request / invalid argument errors from get_library_tracks() and get_liked_songs().  Have tried calling without any passed parameters, or modified parameters, but no calls seem to work.  Tried looking to see if this is a known issue but didn't see reports.  Running latest ytmusicapi version & refreshed OAuth token just in case but no change.  Thanks!

**Comment by thescovells** [comment]
Similar issue here. Been fine for months. Updated to latest version, refreshed oauth token and still the error for any authenticated call. 

<img width="769" alt="Screenshot 2024-11-13 at 19 48 29" src="https://github.com/user-attachments/assets/c4e923f0-1954-45ed-bca1-1db587c2c9d2">

(reverted to the most basic example from the docs here to ensure it wasn't anything else in the code)

**Reply by sigma67** [reply]
please track over at https://github.com/sigma67/ytmusicapi/issues/676

**Comment by sigma67** [comment]
please track over at https://github.com/sigma67/ytmusicapi/issues/676
